tests/SprykerEcoTest/Zed/PunchoutCatalogs/Business/PunchoutCatalogsFacadeTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerTest\Zed\PunchoutCatalogs\Business;
+
+use Codeception\Test\Unit;
+use Generated\Shared\Transfer\CompanyBusinessUnitTransfer;
+use Generated\Shared\Transfer\CompanyTransfer;
+use Generated\Shared\Transfer\CompanyUserTransfer;
+use Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer;
+use Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnection;
+use SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface;
+
+/**
+ * Auto-generated group annotations
+ * @group SprykerTest
+ * @group Zed
+ * @group PunchoutCatalogs
+ * @group Business
+ * @group Facade
+ * @group PunchoutCatalogsFacadeTest
+ * Add your own group annotations below this line
+ */
+class PunchoutCatalogsFacadeTest extends Unit
+{
+    protected const NOT_EXISTING_CONNECTION_ID = '21';
```

Comment:
Are you sure, that you will never have CONNECTION_ID = 21?
I mean, it's always strange to set IDs, that theoretically can be possible.
Can you use `0` here? (it's not possible to have an entity with such ID in all cases).